boards.txt

```diff
@@ -13299,6 +13299,185 @@ adafruit_feather_esp32s3_reversetft.menu.ZigbeeMode.zczr=Zigbee ZCZR (coordinato
+
+adafruit_feather_esp32c6.menu.FlashFreq.80=80MHz
+adafruit_feather_esp32c6.menu.FlashFreq.80.build.flash_freq=80m
+adafruit_feather_esp32c6.menu.FlashFreq.40=40MHz
+adafruit_feather_esp32c6.menu.FlashFreq.40.build.flash_freq=40m
+
+adafruit_feather_esp32c6.menu.UploadSpeed.921600=921600
+adafruit_feather_esp32c6.menu.UploadSpeed.921600.upload.speed=921600
+adafruit_feather_esp32c6.menu.UploadSpeed.115200=115200
+adafruit_feather_esp32c6.menu.UploadSpeed.115200.upload.speed=115200
+adafruit_feather_esp32c6.menu.UploadSpeed.256000.windows=256000
+adafruit_feather_esp32c6.menu.UploadSpeed.256000.upload.speed=256000
+adafruit_feather_esp32c6.menu.UploadSpeed.230400.windows.upload.speed=256000
+adafruit_feather_esp32c6.menu.UploadSpeed.230400=230400
+adafruit_feather_esp32c6.menu.UploadSpeed.230400.upload.speed=230400
+adafruit_feather_esp32c6.menu.UploadSpeed.460800.linux=460800
+adafruit_feather_esp32c6.menu.UploadSpeed.460800.macosx=460800
+adafruit_feather_esp32c6.menu.UploadSpeed.460800.upload.speed=460800
+adafruit_feather_esp32c6.menu.UploadSpeed.512000.windows=512000
+adafruit_feather_esp32c6.menu.UploadSpeed.512000.upload.speed=512000
+
+adafruit_feather_esp32c6.menu.DebugLevel.none=None
+adafruit_feather_esp32c6.menu.DebugLevel.none.build.code_debug=0
+adafruit_feather_esp32c6.menu.DebugLevel.error=Error
+adafruit_feather_esp32c6.menu.DebugLevel.error.build.code_debug=1
+adafruit_feather_esp32c6.menu.DebugLevel.warn=Warn
+adafruit_feather_esp32c6.menu.DebugLevel.warn.build.code_debug=2
+adafruit_feather_esp32c6.menu.DebugLevel.info=Info
+adafruit_feather_esp32c6.menu.DebugLevel.info.build.code_debug=3
+adafruit_feather_esp32c6.menu.DebugLevel.debug=Debug
+adafruit_feather_esp32c6.menu.DebugLevel.debug.build.code_debug=4
+adafruit_feather_esp32c6.menu.DebugLevel.verbose=Verbose
+adafruit_feather_esp32c6.menu.DebugLevel.verbose.build.code_debug=5
+
+adafruit_feather_esp32c6.menu.EraseFlash.none=Disabled
+adafruit_feather_esp32c6.menu.EraseFlash.none.upload.erase_cmd=
+adafruit_feather_esp32c6.menu.EraseFlash.all=Enabled
+adafruit_feather_esp32c6.menu.EraseFlash.all.upload.erase_cmd=-e
+
+adafruit_feather_esp32c6.menu.ZigbeeMode.default=Disabled
+adafruit_feather_esp32c6.menu.ZigbeeMode.default.build.zigbee_mode=
+adafruit_feather_esp32c6.menu.ZigbeeMode.default.build.zigbee_libs=
+adafruit_feather_esp32c6.menu.ZigbeeMode.ed=Zigbee ED (end device)
+adafruit_feather_esp32c6.menu.ZigbeeMode.ed.build.zigbee_mode=-DZIGBEE_MODE_ED
+adafruit_feather_esp32c6.menu.ZigbeeMode.ed.build.zigbee_libs=-lesp_zb_api_ed -lesp_zb_cli_command -lzboss_stack.ed.trace -lzboss_stack.ed -lzboss_port
+adafruit_feather_esp32c6.menu.ZigbeeMode.zczr=Zigbee ZCZR (coordinator)
+adafruit_feather_esp32c6.menu.ZigbeeMode.zczr.build.zigbee_mode=-DZIGBEE_MODE_ZCZR
+adafruit_feather_esp32c6.menu.ZigbeeMode.zczr.build.zigbee_libs=-lesp_zb_api_zczr -lesp_zb_cli_command -lzboss_stack.zczr.trace -lzboss_stack.zczr -lzboss_port
```

Comment:
Please remove the -lzboss_stack.zczr.trace from linker, as it's no longer available.
```suggestion
adafruit_feather_esp32c6.menu.ZigbeeMode.zczr.build.zigbee_libs=-lesp_zb_api_zczr -lesp_zb_cli_command -lzboss_stack.zczr -lzboss_port
```